src/components/Dashboard/LegalObligation.vue

```diff
@@ -81,22 +81,29 @@
 </template>
 
 <script lang="ts">
-import { Component, Mixins } from 'vue-property-decorator'
+import { Component, Mixins, Watch } from 'vue-property-decorator'
 import { Getter } from 'pinia-class'
 import ResourceLookupMixin from '@/mixins/resource-lookup-mixin'
-import { useRootStore } from '@/stores'
+import { useBusinessStore, useRootStore } from '@/stores'
 
 @Component({})
 export default class LegalObligation extends Mixins(ResourceLookupMixin) {
+  @Getter(useBusinessStore) isActive!: boolean
   @Getter(useRootStore) isBusinessWithNoMaintenanceFilings!: boolean
 
   readMoreFlag = false
-  showLegalObligation = true
+  showLegalObligation = null as boolean
```

Comment:
I did this so that it's neither True nor False (for possible debugging), until it's set in the watcher below.

Note that I couldn't just write `showLegalObligation = this.isActive` here because that's evaluated before isActive has its actual value.

I could have set showLegalObligation in a `mounted()` method but I think the watcher design is more accurate as it keeps the value synced (instead of setting it just once).